server/app/assets/javascripts/uswds/uswds-init.min.js

```diff
@@ -0,0 +1,19 @@
+!(function () {
+  'use strict'
+  var n,
+    e = 'usa-js-loading'
+  function t() {
+    document.documentElement.classList.remove(e)
+  }
+  document.documentElement.classList.add(e),
+    (n = setTimeout(t, 8e3)),
+    window.addEventListener(
+      'load',
+      function e() {
+        window.uswdsPresent &&
+          (clearTimeout(n), t(), window.removeEventListener('load', e, !0))
+      },
+      !0,
+    )
+})()
+//# sourceMappingURL=uswds-init.min.js.map
```

Comment:
We discussed offline and this code is directly copied from USWDS, so I think leaving this line in is fine. We think it's meant to tell us where the map gets written to but we're not sure. 

Melanie opted to copy this code here instead of including it in the bundle since it's so small and likely won't change much, which I think is fine. Also the USWDS styles seem to load fine without it. She will add a comment with a link to where the source code for this file lives for context.

Reply:
I was curious about that commented-out line as well.  First things first, what is that map file that Webpack outputs alongside every bundled file?  The map file is the "decoder ring" for the minified JS file.  It allows the browser dev tools to show you the un-minified JS for debugging.  When Webpack creates the map file, it inserts this comment at the end of the related minified JS file so that the browser knows where to find the source map.

Reply:
Very cool! Thanks for explaining and adding the comments in!